pkg/capabilities/consensus/ocr3/types/encoder.go

```diff
@@ -0,0 +1,11 @@
+package types
```

Comment:
Why did this file disappear from the repo? :0

Reply:
That was me... I must have not moved it when I was rebasing my PR and gotten it lost that way.